mesheryctl/pkg/utils/relationship.go

```diff
@@ -93,6 +94,9 @@ func (mrh *RelationshipCSVHelper) ParseRelationshipsSheet(modelName string) erro
 			if modelName != "" && data.Model != modelName {
 				continue
 			}
+			if utils.ReplaceSpacesAndConvertToLowercase(data.PublishToRegistry) == "false" {
```

Comment:
@jerensl, at first glance, it seems like this comparison logic would be set to `!= "false"`, but I could be looking at this out of context. So long as we're no longer publishing Relationship Definitions with "publishToRegistry" of "false", we're good to go in this PR.

P.S. A small code comment for this `if` condition might be helpful.